packages/rules-unit-testing/test/database.test.ts

```diff
@@ -128,6 +128,31 @@ describe('Testing Module Tests', function () {
       .catch(() => {});
   });
 
+  it('assertFails() if code is Permission denied', async function () {
+    const success = Promise.resolve('success');
+    const permissionDenied = Promise.reject({
+      code: 'Permission denied'
```

Comment:
```suggestion
      message: 'Permission denied'
```

This fails on CI: https://github.com/firebase/firebase-js-sdk/pull/4668/checks?check_run_id=2169950419

I think you'd need `message` instead of `code` here.

Reply:
Whoops! I made the fix.

@google-oss-bot I signed it.